src/widgets/calendar.rs

```diff
@@ -146,16 +146,16 @@ impl<'a, S: DateStyler> Widget for Monthly<'a, S> {
 
         // go through all the weeks containing a day in the target month.
         while curr_day.month() as u8 != self.display_date.month().next() as u8 {
-            let mut line = Spans(Vec::with_capacity(14));
+            let mut line = Spans::from(Vec::with_capacity(14));
             for i in 0..7 {
                 // Draw the gutter. Do it here so we can avoid worrying about
                 // styling the ' ' in the format_date method
                 if i == 0 {
-                    line.0.push(Span::styled(" ", Style::default()));
+                    line.content.push(Span::styled(" ", Style::default()));
```

Comment:
I don't think so, because most consumers will not use that method anyways. Similarly to `Spans::from` most users will probably rely on that + putting together their `Vec<Span>` beforehand.

It would be a quite easy addition later if it is ever deemed worth the addition.